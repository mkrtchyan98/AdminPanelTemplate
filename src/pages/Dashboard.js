import { Helmet } from 'react-helmet';
import { Box, Container, Grid } from '@material-ui/core';

const Dashboard = () => (
  <>
    <Helmet>
      <title>Dashboard | Admin</title>
    </Helmet>
    <Box
      sx={{
        backgroundColor: 'background.default',
        minHeight: '100%',
        py: 3
      }}
    >
      <Container maxWidth={false}>
        <Grid container spacing={3}>
          <Grid item lg={3} sm={6} xl={3} xs={12}>
            Here will be something
          </Grid>
        </Grid>
      </Container>
    </Box>
  </>
);

export default Dashboard;
