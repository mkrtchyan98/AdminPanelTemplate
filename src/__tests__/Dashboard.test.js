import React from 'react';
import { mount } from 'enzyme';
import Dashboard from '../pages/Dashboard';

describe('render Dashboard component', () => {
  it('renders snapshot', () => {
    const wrapper = mount(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
  });
});
