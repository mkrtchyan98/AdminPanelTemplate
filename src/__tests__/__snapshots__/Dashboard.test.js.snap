// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Dashboard component renders snapshot 1`] = `ShallowWrapper {}`;
